lib/server/services/shift/ShiftService.js

```diff
@@ -58,40 +58,43 @@ class ShiftService {
      * @param {Date} shiftStart the start time of the shift (to check for previous information)
      * @return {Promise<object>} the shift data
      */
-    async getShiftData(shifterUserIdentifier, shiftType, shiftStart = null) {
+    async getShiftData(shifterUserIdentifier, shiftType) {
         const shift = await this.getUserPendingShiftOrFail(shifterUserIdentifier);
 
         const ret = {
             shift,
             issuesLogs: await this.getShiftIssues(shift.start, shiftType),
             typeSpecific: null,
-            previousInfo: { value: '' },
+            infoFromPreviousShifter: { value: '' },
         };
 
         // Find EoS reports from the last shift with the correct title
-        const timestamp = shiftStart || shift.start;
-        const pastShift = getShiftFromTimestamp(timestamp - SHIFT_DURATION);
+        const pastShift = getShiftFromTimestamp(shift.start - SHIFT_DURATION);
         const title = formatEosReportTitle(pastShift, shiftType);
 
-        const prevShiftIssues = await getShiftIssuesByTitle(pastShift, title);
+        const prevShiftIssues = await getShiftIssuesByTitle(title);
```

Comment:
Please name variables in a more informative way. I think title should be `expectedPreviousReportTitle` and previousShiftIssues should be `previousShiftReports`